Makefile

```diff
@@ -230,7 +230,7 @@ format:
 
 lint:
 	@echo "--> Running linter"
-	@go run github.com/golangci/golangci-lint/cmd/golangci-lint@latest run
```

Comment:
lint fail when golangci-lint bump from [v1.52.2 ](https://github.com/cometbft/cometbft/actions/runs/5135523040/jobs/9241058231) to [v1.53.1](https://github.com/cometbft/cometbft/actions/runs/5163633639/jobs/9302427839?pr=865), for more [info](https://github.com/golangci/golangci-lint/compare/v1.52.2...v1.53.2)